rhcos4/profiles/moderate.profile

```diff
@@ -1,5 +1,11 @@
 documentation_complete: true
 
+metadata:
+    policy_hub: https://www.fedramp.gov/documents/
```

Comment:
Are you sure? This is a moderate profile. Is it FISMA Moderate? FedRAMP moderate? etc.

Reply:
You see? The maintainers know better about their stuff, :laughing:.
Hopefully it is correct now.